KTLOSAQ40Loot.json

```diff
@@ -547,7 +547,7 @@
 			"wowID":"20933"
 		},
 		"Carapace of the Old God":{
-			"prio":["Tanks","r12+ Hunter>Rogue>Ret>Warrior","EP/GP"],
+			"prio":["Shadowsfade(1)","Killstep(2)","Chart(1)","Stopbegging(2)","Supertoxic(1)","BenDriller(1)","Tanks","r12+ Hunter>Rogue>Ret>Warrior","EP/GP"],
```

Comment:
oph should be first on this right? Id also like to be in there somewhere, after the main rogues, just as a heads up, after super or you

Reply:
I've updated him to be first. Will you use the chest for DPSing or only when you're tanking? If it's dual use you can have it before me, if not I'll probably add you after me. But maybe still before me. I like using darkmantle but I also think I'll probably have to run t2.5 because having no stamina is a real killer.